pkg/clusterclientstore/interface.go

```diff
@@ -7,13 +7,13 @@ import (
 )
 
 type ClientProvider interface {
-	GetClient(clusterName string) (kubernetes.Interface, error)
-	GetConfig(clusterName string) (*rest.Config, error)
+	GetClient(string, string) (kubernetes.Interface, error)
```

Comment:
I would suggest keeping the variable names in the interface method signatures as this gives an idea what these variables are without checking out a specific implementation.